c_glib/arrow-glib/type.h

```diff
@@ -115,8 +116,8 @@ typedef enum {
   GARROW_TYPE_LARGE_LIST,
   GARROW_TYPE_MONTH_DAY_NANO_INTERVAL,
   GARROW_TYPE_RUN_END_ENCODED,
-  /* TODO: Remove = 44 when we add STRING_VIEW..DECIMAL32. */
-  GARROW_TYPE_DECIMAL64 = 44,
```

Comment:
We can't remove this.
If we remove this, `GARROW_TYPE_DECIMAL32` is `39` and `GARROW_TYPE_DECIMAL64` is `40`.

Reply:
Thanks fixed. I didn't understand what `STRING_VIEW` meant. Oh, It is the data type.